packages/react-components/src/organisms/MembersSection.tsx

```diff
@@ -1,72 +1,72 @@
 import React from 'react';
 import css from '@emotion/css';
+import { TeamMember } from '@asap-hub/model';
 
-import { Link, Card, Headline2, Avatar, Paragraph } from '../atoms';
-import { perRem, mobileScreen } from '../pixels';
-
-type MembersSectionProps = {
-  readonly members: {
-    readonly id: string;
-    readonly firstName?: string;
-    readonly lastName?: string;
-    readonly displayName: string;
-    readonly role: string;
-    readonly avatarURL?: string;
-  }[];
-};
+import { Link, Card, Headline2, Avatar } from '../atoms';
+import { lead } from '../colors';
+import { perRem, tabletScreen } from '../pixels';
 
 const containerStyles = css({
+  margin: 0,
   padding: 0,
-  listStyle: 'none',
+  display: 'grid',
 
-  display: 'flex',
-  flexDirection: 'row',
-  flexWrap: 'wrap',
-});
+  gridColumnGap: `${18 / perRem}em`,
+  gridTemplateColumns: 'min-content 1fr',
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    gridTemplateColumns: 'min-content 1fr min-content 1fr',
+  },
 
-const avatarContainer = css({
-  flexBasis: 1,
+  alignItems: 'center',
+  gridAutoRows: '1fr',
+  gridAutoFlow: 'row dense',
 });
 
-const detailsContainer = css({
-  paddingLeft: `${12 / perRem}em`,
+const avatarStyles = css({
+  gridRowEnd: 'span 2',
 });
+const nameStyles = css({
+  alignSelf: 'end',
+  marginTop: `${12 / perRem}em`,
 
-const memberContainerStyles = css({
-  flexBasis: '50%',
-  [`@media (max-width: ${mobileScreen.width}px)`]: {
-    flexBasis: '100%',
-    flexDirection: 'column',
+  fontWeight: 'bold',
+});
+const roleStyles = css({
+  gridColumn: 2,
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    'li:nth-of-type(2n) &': {
```

Comment:
I didn't realize `emotion` supported this (referring to parent elements by putting `&` at the end, this opens up amazing possibilities...)